metrics/register.go

```diff
@@ -0,0 +1,24 @@
+package metrics
+
+import (
+	"fmt"
+
+	"go.opencensus.io/stats/view"
+)
+
+var registeredViews = map[string][]view.View{}
+
+func RegisterViews(name string, views []view.View) {
```

Comment:
likewise let's return an error rather than panicking here. i've thought it over and i think that's best!